Lib/glyphsLib/classes.py

```diff
@@ -1297,8 +1298,8 @@ def width(self, value):
 
 
 class GSNode(GSBase):
-    _rx = '([-.e\d]+) ([-.e\d]+) (LINE|CURVE|QCURVE|OFFCURVE|n/a)'\
-          '(?: (SMOOTH))?(?: (\{.*\}))?'
+    _rx = '"([-.e\d]+) ([-.e\d]+) (LINE|CURVE|QCURVE|OFFCURVE|n/a)'\
+          '(?: (SMOOTH))?(?: (\{(?:.|\\n)*\}))?"'
```

Comment:
`re.DOTALL` flag makes the dot also match newlines
https://docs.python.org/3.7/library/re.html#re.DOTALL